frontend/javascripts/test/snapshots/public/test-bundle/test/backend-snapshot-tests/projects.e2e.js.md

```diff
@@ -467,80 +467,60 @@ Generated by [AVA](https://avajs.dev).
       tracingTime: 0,
     }
 
-## projects-pauseProject(projectName: string)
+## projects-pauseProject(projectId: string)
 
     {
       expectedTime: 90,
-      id: '58135bfd2faeb3190181c057',
+      id: '68135bfd2faeb3190181c058',
       isBlacklistedFromReport: false,
-      name: 'Test_Project',
+      name: 'Test_Project2',
```

Comment:
Hmm I don’t think it should have. Yes I guess plugging in the ID directly in the test may be better. But I’ll also have another look at why the ordering may have changed.

Reply:
Turns out getProjects orders them by creation date. In the test db, all had the same creation date so the ordering was in fact not deterministic. I changed the test db, so that should be fixed now :)

Reply:
Awesome! Thanks for investigating :+1: